Signal/src/call/Speakerbox/CallKitCallUIAdaptee.swift

```diff
@@ -234,6 +242,19 @@ final class CallKitCallUIAdaptee: NSObject, CallUIAdaptee, CXProviderDelegate {
         _ = self.callService.handleOutgoingCall(call)
         action.fulfill()
         self.provider.reportOutgoingCall(with: call.localId, startedConnectingAt: nil)
+
+        ensureCallName(call:call)
```

Comment:
nit: This function name is a little funny, and we're only using it in one place. I'd prefer just inlining this function. But if you want to keep it you should call it something that hints we're reporting the call, not just munging the name.